rider-fsharp/build.gradle.kts

```diff
@@ -262,6 +265,10 @@ tasks {
 
   named<Test>("test") {
     dependsOn(parserTest)
+    classpath -= classpath.filter {
+      (it.name.startsWith("localization-") && it.name.endsWith(".jar")) // TODO[#478]: https://youtrack.jetbrains.com/issue/IJPL-178084/External-plugin-tests-break-due-to-localization-issues
+        || it.name == "cwm-plugin.jar" // TODO[#479]: Check after 251 EAP5 release
```

Comment:
Could you check it please? 🙂

Reply:
We fixed it just recently, yes.
https://github.com/ForNeVeR/AvaloniaRider/commit/eac36a3cd5d88713b14ee4ca528d1f0fb15b528a